### ISSUE TITLE
Toggle expand/collapse by clicking test result name

### ISSUE THREAD

**Issue by cynicaloptimist** [PeakyTests/Peaky#25]
This might be a little easier to use than having to click the little [+] [-] buttons.

**Comment by jonsequitur** [comment]
Agreed, but I wouldn't want to remove the +/- buttons, since those are more discoverable.

**Comment by PhillipPruett** [comment]
Yeah I found myself wanting the title to collapse the result. It would be a nice convenience.
